pkg/kubectl/namespace_test.go

```diff
@@ -43,19 +44,59 @@ func TestNamespaceGenerate(t *testing.T) {
 		{
 			params:    map[string]interface{}{},
 			expectErr: true,
+			index:     1,
+		},
+		{
+			params: map[string]interface{}{
+				"name": 1,
+			},
+
+			expectErr: true,
+			index:     2,
+		},
+		{
+			params: map[string]interface{}{
+				"name": nil,
+			},
+
+			expectErr: true,
+			index:     3,
+		},
+		{
+			params: map[string]interface{}{
+				"name_wrong_key": "some_value",
+			},
+
+			expectErr: true,
+			index:     4,
+		},
+		{
+			params: map[string]interface{}{
+				"NAME": "some_value",
+			},
+
+			expectErr: true,
+			index:     5,
 		},
 	}
 	generator := NamespaceGeneratorV1{}
 	for _, test := range tests {
```

Comment:
`for index, test := range tests {` ought to do what you want without specifying it in each test.